internal/setup/slices_db.go

```diff
@@ -0,0 +1,73 @@
+package setup
+
+import (
+	"bufio"
+	"os"
+	"path"
+	"path/filepath"
+	"reflect"
+)
+
+const slicesDBPath = "/var/lib/chisel/chisel.db"
+
+// Read and store the slices that have been already installed
+var installedSlices []SliceKey
```

Comment:
This would be better like this:

```
type ChiselDB struct {
    installedSlices []SliceKey
}
```

Then you'd have

```
func ReadDB(rootdir string) (*ChiselDB, error) {
    // same as ReadInstalledSlices
}

func (db *ChiselDB) IsSliceInstalled(slice SliceKey) bool {
    // same as IsSliceInstalled except refer to db.installedSlices instead of installedSlices
}

func (db *ChiselDB) WriteInstalledSlices(rootdir string, slices []*Slice) (error) {
    // same as WriteInstalledSlices except refer to db.installedSlices instead of installedSlices
    // also append to db.installedSlices
}

```